test/test_config_parser.rb

```diff
@@ -53,6 +37,20 @@ def test_multiple
     assert(ConfigParser.proto_config(ConfigParser.parse_config('test/data/multiple.conf')) == multiple)
   end
 
+  # test to make sure embedded ruby is not parsed
+  def test_emb
```

Comment:
General comment: Give each test a descriptive name, and then omit the comments.

For example, this test could be:

    def test_embedded_ruby_does_not_get_parsed

Reply:
Good suggestion! Will make that change.